File: src/main/java/domain/Agency.java
package domain;

public class Agency implements Identifiable<Integer> {
    private int id;
    private final String name;
    private String password;

    public Agency(String name, String password) {
        this.name = name;
        this.password = password;
    }

    public Agency(int id, String name, String password) {
        this.id = id;
        this.name = name;
        this.password = password;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }
}

File: src/domain/Agency.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace AgentiiDeTurism.src.domain
{
    public class Agency : IIdentifiable<int>
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Password { get; set; }
        
        public Agency(string name, string password)
        {
            this.Name = name;
            this.Password = password;
        }

        public Agency(int id, string name, string password)
        {
            this.Id = id;
            this.Name = name;
            this.Password = password;
        }

        public int getId()
        {
            return this.Id;
        }

        public void setId(int id)
        {
            this.Id = id;
        }

        public override string ToString()
        {
            return this.Name;
        }
    }
}
